cmd/juju/controller/configcommand.go

```diff
@@ -0,0 +1,232 @@
+// Copyright 2016 Canonical Ltd.
```

Comment:
2018?

Reply:
I renamed getconfig.go to configcommand.go - I'm not sure whether I should update the date in that case?